### PR TITLE
[Bug] Fixed download file for microsoft edge

### LINKED ISSUE #1069: [Bug] Not able to save or export map (Windows/MSEdge)
**Describe the bug**
I cannot save or export the map created in Kepler.

**To Reproduce**
1. Go to a created map in Kepler.gl
2. Click on 'Save', click 'login' under Dropbox
3. See error. A new blank project window popup.
or
1. Click 'Share', then click any of the four options.
2. Select the selections provided.
3. Click 'Download' or 'Export'.
4. See error. No response.

**Expected behavior**
Successfully save or export the map.

**Screenshots**
image
image
image

image
image
image
image
image

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]  (not OS) Windows 10
 - Browser [e.g. chrome, safari] Microsoft Edge
 - Version [e.g. 22] 44.18362.449.0

**Smartphone (please complete the following information):**

**Additional context**
I'm wondering if the tool is specifically designed for OS system? I cannot use this online tool in Windows, can I?

### PR DESCRIPTION
**Pull request by macrigiuseppe**
This PR fixes #1069 

The new logic detects if user is using a MS edge browser and behaves accordingly

Signed-off-by: Giuseppe Macri <gmacri@uber.com>

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1286,6 +1286,39 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
+"@formatjs/intl-displaynames@^1.2.0":
```
Comment (macrigiuseppe) [inline yarn.lock:1289]: i ran yarn and these changes popped up

File: src/utils/export-image-utils.js [hunk 2]
```diff
@@ -1,74 +0,0 @@
-// Copyright (c) 2020 Uber Technologies, Inc.
```
Comment (heshan0131) [inline src/utils/export-image-utils.js:1]: why is this deleted?

Comment (macrigiuseppe) [reply src/utils/export-image-utils.js:1]: None of the methods was actually used. They were duplicated from export-utils.
